### PR TITLE
Fix md column syntax

### PR DESCRIPTION
**Pull request by heath-hamilton**
Fixes the Markdown syntax issues on the overview page and also recasts the look book link as a call to action in line with the others.

### CODE REVIEW

File: msteams-platform/overview.md [hunk 1]
```diff
@@ -137,7 +131,23 @@ You don't need to be an expert programmer to build a great Teams app. Try one of
    :::column-end:::
 :::row-end:::
 
-## Resources
+:::row:::
+   :::column span="2":::
+
+## Get ideas for your app
+
+Looking for inspiration? Browse our list of real-world scenarios and industry solutions to understand the ways Teams apps can help your users.
```
Comment (sathya-raveendran) [inline msteams-platform/overview.md:139]: Looking for app development inspiration? Browse our list of real-world scenarios and industry solutions with high fidelity concept mocks to understand various ways Teams apps can help your users.

**Review by sathya-raveendran — APPROVED** [review]
Looking for app development inspiration? Browse our list of real-world scenarios and industry solutions with high fidelity concept mocks to understand various ways Teams apps can help your users.

File: msteams-platform/overview.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ You don't need to be an expert programmer to build a great Teams app. Try one of
 
 ## Get ideas for your app
 
-Looking for inspiration? Browse our list of real-world scenarios and industry solutions to understand the ways Teams apps can help your users.
+Looking for app development inspiration? Browse our list of real-world scenarios and industry solutions with high fidelity concept mocks to understand various ways Teams apps can help your users.
```
Comment (heath-hamilton) [inline msteams-platform/overview.md:139]: @KirtiPereira Can we fix the grammar issue by adding "the" between "understand" and "various"? Thank you!

Looking for app development inspiration? Browse our list of real-world scenarios and industry solutions with high fidelity concept mocks to understand the various ways Teams apps can help your users.

**Review by heath-hamilton — COMMENTED** [review]
Please see my one minor comment, @KirtiPereira.

**Review by surbhigupta12 — APPROVED** [review]
Thanks for the updates

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**